packages/app/src/app/overmind/namespaces/pro/actions.ts

```diff
@@ -3,17 +3,31 @@ import { withLoadApp } from 'app/overmind/factories';
 import { SubscriptionInterval } from 'app/graphql/types';
 import { Step, PaymentSummary, PaymentPreview } from './types';
 
-export const getPrices = async ({ state, effects }: Context) => {
+// This exists but doesn't do anything
+export const getLegacyPrices = async ({ state, effects }: Context) => {
   try {
-    state.pro.prices = await effects.api.prices();
+    state.pro.legacyPrices = await effects.api.legacyPrices();
   } catch (err) {
     // Fail silently.
   }
 };
 
+// This exists but doesn't work. Copied from the function above.
+export const getPrices = async ({ state, effects }) => {
+  try {
+    state.pro.prices = await effects.api.getPrices();
+  } catch (e) {
+    // e
+  }
+};
+
 export const pageMounted = withLoadApp(async ({ effects, state, actions }) => {
-  state.pro.prices = await effects.api.prices();
+  // We have to call the api effects directly rather than using an action because
+  // for some reason actions don't work.
```

Comment:
I think I can move the functions `getLegacyPrices` and `getPrices` above, but I'm not 100% sure if overmind does anything else with them. I don't think so.